torch/csrc/jit/mobile/import.cpp

```diff
@@ -341,14 +348,15 @@ c10::IValue BytecodeDeserializer::readArchive(
     return len;
   };
 
-  static const c10::QualifiedName torchPrefix = "__torch__";
+  static const c10::QualifiedName torchPrefix__ = "__torch__";
+  static const c10::QualifiedName torchPrefix = "torch";
```

Comment:
@SplitInfinity and @suo , it looks hacky here to just work around runtime error. The error happens when readArchive("data.pkl"), because the type parser in lite interpreter cannot resolve type with token "torch". data.pkl looks like:
```
torch.jit.mul_test_backendLoweredModule()(state=
 (torch.jit._pickle.restore_type_tag({'forward': torch.jit._pickle.restore_type_tag({'': ''}, 'Dict[str, Any]')},
    'Dict[str, Any]'),
  __torch__.___torch_mangle_0.M()(state={})))
```
Looks like it's used to describe the module itself, but it's not used at lite interpreter runtime. Do you have any better idea to work around?

Reply:
Unfortunately I don't know *anything* about the details of pickling/unpickling 🙉 . I can dig some more next week if @suo doesn't know.